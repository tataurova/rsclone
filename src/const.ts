export const RouteName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};

export const AuthorizationStatus = {
  AUTH: 'AUTH',
  NO_AUTH: 'NO_AUTH',
};

export const Error = {
  UNAUTHORIZED: 401,
  BAD_REQUEST: 400,
  NOT_FOUND: 404,
  SERVICE_UNAVAILABLE: 503,
  TIMEOUT_ERROR: 'timeout_error',
};

export const headCells = [
  { id: 'status', label: 'Статус' },
  { id: 'city', label: 'Город' },
  { id: 'name', label: 'ФИО' },
  { id: 'age', label: 'Возраст' },
  { id: 'date', label: 'Дата' },
  { id: 'coordinator', label: 'Координатор' },
  { id: 'people', label: 'Людей на поиске' },
  { id: 'actions', label: 'Действия', disableSorting: true },
];

export const getStatusCollection = [
  'Резерв',
  'Поиск на месте',
  'Стоп. Проверка информации',
  'Архив',
];

export const SearcherStatus = {
  way: 'В пути',
  place: 'На месте',
  home: 'Дома',
};

export const TransportStatus = {
  car: 'На машине',
  own: 'Своим ходом',
  need: 'Нужна машина',
};

export const MAX_TIMEOUT = 5000;
export const SHOW_ERROR_TIMEOUT = 3000;
