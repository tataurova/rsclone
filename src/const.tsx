export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
