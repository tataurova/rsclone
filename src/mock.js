const data = [
  {
    id: 1,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 2,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Пертов Петр Петрович',
    city: 'Москва',
    birth: '04.09.1985',
    age: '30',
    missing: '01.10.2017',
    sign: 'Был с собакиой',
    close: 'Шорты, белоя майка',
    with: 'черная сумка',
  },
  {
    id: 3,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Кузмин Картой Петрович',
    city: 'Судлино',
    birth: '30.12.1960',
    age: '60',
    missing: '05.12.2018',
    sign: 'Хромает на леаую ногу',
    close: 'Шуба, валенки',
    with: 'трость',
  },
  {
    id: 4,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Щаруго Кирилл Федорович',
    city: 'Пирам',
    birth: '14.07.2000',
    age: '20',
    missing: '05.10.2010',
    sign: 'Заикается',
    close: 'Кепка синяя, белая майка',
    with: 'короткая стрижка',
  },
  {
    id: 5,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 6,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '25.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 7,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    sign: 'Шрам на левой щеке',
    missing: '05.12.2020',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 8,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.04.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 9,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
];

export default data;
