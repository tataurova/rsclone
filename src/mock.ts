export const data = [
  {
    id: 1,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 2,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Пертов Петр Петрович',
    city: 'Москва',
    birth: '04.09.1985',
    age: '30',
    missing: '01.10.2020',
    sign: 'Был с собакиой',
    close: 'Шорты, белая майка',
    with: 'черная сумка',
  },
  {
    id: 3,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Кузмин Картой Петрович',
    city: 'Судлино',
    birth: '30.12.1960',
    age: '60',
    missing: '05.12.2020',
    sign: 'Хромает на левую ногу',
    close: 'Шуба, валенки',
    with: 'трость',
  },
  {
    id: 4,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Щаруго Кирилл Федорович',
    city: 'Пирам',
    birth: '14.07.2000',
    age: '20',
    missing: '05.10.2020',
    sign: 'Заикается',
    close: 'Кепка синяя, белая майка',
    with: 'короткая стрижка',
  },
  {
    id: 5,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2020',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 6,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '25.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 7,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    sign: 'Шрам на левой щеке',
    missing: '05.12.2020',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 8,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.04.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 9,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },

];

export const mockActiveSearches = [
  {
    id: 1,
    status: 'Поиск на месте',
    city: 'Санкт-Петербург',
    name: 'Петр',
    age: '30',
    date: '12-01-2020',
    coordinator: 'Чиж',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }, {
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  }, {
    id: 3,
    status: 'Стоп. Проверка информации',
    city: 'Минск',
    name: 'Аркадий',
    age: '100',
    date: '01-01-2021',
    coordinator: 'Чип',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
  {
    id: 4,
    status: 'Поиск на месте',
    city: 'Москва',
    name: 'Владимир',
    age: '3000',
    date: '12-06-2020',
    coordinator: 'Дейл',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
  {
    id: 5,
    status: 'Резерв',
    city: 'Санкт-Петербург',
    name: 'Петр',
    age: '30',
    date: '12-01-2020',
    coordinator: 'Чи',
    people: [],
  },
  {
    id: 6,
    status: 'Поиск на месте',
    city: 'Киев',
    name: 'Анна',
    age: '123',
    date: '12-01-2020',
    coordinator: 'Кат',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
];
export const mockClosedSearches = [{
  id: 2,
  status: 'Архив',
  city: 'Колпино',
  name: 'Фекла',
  age: '200',
  date: '10-01-2020',
  coordinator: 'Чиж',
  people: [{
    name: 'Саша',
    status: 'way',
    transport: 'car',
    places: '1',
    time: '12:00',
    district: 'Василеостровский',
  }, {
    name: 'Катя',
    status: 'place',
    transport: 'own',
    places: '0',
    time: '19:00',
    district: 'Василеостровский',
  }, {
    name: 'Рома',
    status: 'place',
    transport: 'need',
    places: '0',
    time: '2:00',
    district: 'Калининский',
  }, {
    name: 'Илья',
    status: 'way',
    transport: 'own',
    places: '0',
    time: '15:00',
    district: 'Выборгский',
  }],
}];
