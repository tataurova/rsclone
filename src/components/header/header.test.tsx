import * as React from 'react';
import * as renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { configure } from 'enzyme';
import { Provider } from 'react-redux';
import * as Adapter from 'enzyme-adapter-react-16';
import Header from './header';
import { mockActiveSearches, mockClosedSearches } from '../../mock';
import { AuthorizationStatus } from '../../const';

configure({ adapter: new Adapter() });

describe('<Header />', () => {
  it('Render Header without auth', () => {
    const mockStore = configureStore([]);
    const store = mockStore({
      DATA: {
        gonePeople: [],
        lookingRelativesPeople: [],
        activeSearches: mockActiveSearches,
        closedSearches: mockClosedSearches,
        isFetching: false,
        error: false,
      },
      APP: {
        page: '2',
      },
      AUTH: {
        authorizationStatus: AuthorizationStatus.NO_AUTH,
        user: '',
      },
    });

    const tree = renderer
      .create(
                <Provider store={store}>
                    <Header
                        page={'0'}
                        authorizationStatus={AuthorizationStatus.NO_AUTH}
                    />
                </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('Render Header with auth', () => {
    const mockStore = configureStore([]);
    const store = mockStore({
      DATA: {
        gonePeople: [],
        lookingRelativesPeople: [],
        activeSearches: mockActiveSearches,
        closedSearches: mockClosedSearches,
        isFetching: false,
        error: false,
      },
      APP: {
        page: '2',
      },
      AUTH: {
        authorizationStatus: AuthorizationStatus.AUTH,
        user: '',
      },
    });

    const tree = renderer
      .create(
                <Provider store={store}>
                    <Header
                        page={'0'}
                        authorizationStatus={AuthorizationStatus.AUTH}
                    />
                </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
