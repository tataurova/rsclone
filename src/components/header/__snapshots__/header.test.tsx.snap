// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Render Header with auth 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <button
        aria-label="menu"
        className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-2 MuiIconButton-colorInherit MuiIconButton-edgeStart"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            onClick={[Function]}
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </button>
      <h6
        className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
      >
        Main
      </h6>
      <img
        alt="Logo"
        className="makeStyles-img-4"
        height="50px"
        src=""
        width="50px"
      />
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Logout
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </button>
    </div>
  </header>
</div>
`;

exports[`<Header /> Render Header without auth 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <button
        aria-label="menu"
        className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-2 MuiIconButton-colorInherit MuiIconButton-edgeStart"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            onClick={[Function]}
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
      </button>
      <h6
        className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
      >
        Main
      </h6>
      <img
        alt="Logo"
        className="makeStyles-img-4"
        height="50px"
        src=""
        width="50px"
      />
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Login
        </span>
      </button>
    </div>
  </header>
</div>
`;
